test/python/apps/bitcoin.py

```diff
@@ -59,7 +59,7 @@ class Command:
 
 
 class BitcoinClient:
-    CLA = 0xE0
+    CLA = 0xE1
```

Comment:
Why this change ? @lpascal-ledger

Reply:
`0xE0` was the CLA for the old Bitcoin application. The new one is now `0xE1`. This does not impact anything as currently there are no communication directly with the BTC application in the Python tests. It's more a reminder for possible future use (although we would probably use [the official client](https://pypi.org/project/ledger-bitcoin/) instead of this code).

Reply:
It appears that `BitcoinClient` is not actually used, so probably better to remove it?
Just changing `E0` to `E1` wouldn't fix the class, as the protocol is completely different anyway.

Reply:
Indeed, it's dead code, I'll remove it.